app/Http/Controllers/OrderLineController.php

```diff
@@ -60,6 +62,11 @@ public function index($date = null)
             }
         }
 
+        $outstanding = Activity::whereHas('users', function (Builder $query) {
```

Comment:
Could you also add the remaining amount of other products?

Reply:
I just tried that but think it is confusing because they will be under the 'next withdrawal' amount. This amount will only be things that will not be in the next withdrawal which the other products always are.

Reply:
The only thing that is not under the next withdrawal en will be subtracted later are unclosed events which will be in this amount.

Reply:
Fair